src/components/404.vue

```diff
@@ -0,0 +1,373 @@
+        Oops...
+        <br />Page Not Found 🤔
+      </div>
+      <div id="back_to_home">Back to home</div>
+    </div>
+    <div id="bottom_links">
+      <div id="copyright">Cornell DTI @ 2019</div>
+      <div id="love">Built with LOVE</div>
+      <div id="privacy">Privacy Policy</div>
+    </div>
+  </body>
+</html>
+</template>
+
+<script>
+export default {
+  methods: {
+    placeholder() {
+      const boxes = document
+        .getElementById('band')
+        .querySelectorAll('.color_box');
+
+      for (let x = 0; x < boxes.length; x += 1) {
+        const posy = parseInt(boxes[x].offsetTop, 10) - 55.16;
+        boxes[x].style.top = posy;
+      }
+    }
+  }
+};
+</script>
+
+
+<style scoped lang="scss">
+.band {
+  width: 100%;
+  height: 60%;
+}
+.color_box {
+  /* margin: 2px; */
+  display: inline;
+}
+
+/* #first_four{
+    height: 143.32px;
+    display: inline-block;
+} */
+/* start first four */
+#first_box1 {
+  position: absolute;
+  width: 34.72px;
+  height: 72px;
+  left: 226.75px;
+  top: 215.16px;
+  background: #92c3e6;
+  border-radius: 1.3px;
+  transform: skew(-25deg);
+}
+#second_box1 {
+  position: absolute;
+  width: 28px;
+  height: 72px;
+  left: 281.26px;
+  top: 215.16px;
+  background: #1aa9a5;
+}
+#third_box1 {
+  position: absolute;
+  width: 25px;
+  height: 109px;
+  left: 251.75px;
+  top: 249.16px;
+  background: #508197;
+  transform: rotate(90deg);
+}
+#fourth_box1 {
+  position: absolute;
+  width: 28px;
+  height: 23px;
+  left: 281.26px;
+  top: 320.16px;
+  background: #105351;
+}
+/* end first four */
+/* start zero */
+#first_box2 {
+  position: absolute;
+  width: 81.6px;
+  height: 33.6px;
+  left: 352.83;
+  top: 215.16px;
+  background: #92c3e6;
+}
+#second_box2 {
+  position: absolute;
+  width: 33.6px;
+  height: 81.6px;
+  left: 439.23;
+  top: 215.16px;
+  background: #1aa9a5;
+}
+#third_box2 {
+  position: absolute;
+  width: 33.6px;
+  height: 81.6px;
+  left: 352.83;
+  top: 253.56;
+  background: #508197;
+}
+#fourth_box2 {
+  position: absolute;
+  width: 81.6px;
+  height: 33.6px;
+  left: 391.23;
+  top: 301.56;
+  background: #105351;
+}
+/* end zero */
+/* start second four 313.26 is diff*/
+#first_box3 {
+  position: absolute;
+  width: 34.72px;
+  height: 72px;
+  left: 525px;
+  top: 215.16px;
+  background: #92c3e6;
+  border-radius: 1.3px;
+  transform: skew(-25deg);
+}
+#second_box3 {
+  position: absolute;
+  width: 28px;
+  height: 72px;
+  left: 579.52px;
+  top: 215.16px;
+  background: #1aa9a5;
+}
+#third_box3 {
+  position: absolute;
+  width: 25px;
+  height: 109px;
+  left: 550px;
+  top: 249.16px;
+  background: #508197;
+  transform: rotate(90deg);
+}
+#fourth_box3 {
+  position: absolute;
+  width: 28px;
+  height: 23px;
+  left: 579.52px;
+  top: 320.16px;
+  background: #105351;
+}
+/* end second four */
+
+#oops {
+  position: absolute;
+  width: 553px;
+  height: 134px;
+  left: 670px;
+  top: 207px;
+  font-family: Proxima Nova;
```

Comment:
I know Figma has font-family's like this, but it is a bit more complicated than this. We had to import font files to CoursePlan ourselves, and we set font-family in _global.scss. Check this file if you want to see how we set up the font families. You never need to set the font-family attribute in any other css file in our code because we set it globally in that file, so just delete this line (and the other ones throughout this file).